open-metadata-implementation/user-interfaces/presentation-server/README.md

```diff
@@ -54,6 +46,16 @@ the types of [Egeria OMAG Servers](../../admin-services/docs/concepts/omag-serve
   then the browser url to use to login is `https://localhost:8091/aaa`. All browser requests for this tenant will be issued using urls starting
   `https://localhost:8091/aaa`. 
     
+## FOR NEW DEVELOPMENT (as of 8/25/2020)
+ * Currently, the UI is not optimized to run in development mode. Developers must rebuild the static files every time a change is made.
+ * Because of this, work is currently underway to remake the client with Create React App. This will be done iteratively, and updates posted here.
+ * Old client code will be maintained until the cutover is complete, so developers can run the traditional way if desired.
+ * The new client is maintained in the cra-client directory.
+ * The new server is maintained in the cra-server directory.
+ * Set up your environment by running `npm install` in both cra-client and cra-server.
+ * Start the new UI by navigating into the cra-server directory, then running `npm start`.
+ * Enter `https://localhost:3000/` to access the new UI in development mode.
+ * Due to https, was only able to get it running properly using Firefox browser (no Chrome).
```

Comment:
What issues did you get? I will try later... it may be due to the way current versions of chrome do further validation. We hit some issues even for the base platform. Whilst the config work isn't complete, it does work (in chrome) and notes are at #1403. Specifically it could relate to the certificate you've used. 

One option -- if we don't fix during PR, open up a new issue & fix it there

Reply:
We have a separate conversation regarding this in #3543, so will move topic to there.